manifests/exporter.pp

```diff
@@ -54,33 +54,21 @@
   }
 
 
-
   ######################2. Metrics Dash Board deployment ###############
   # Assuming use of puppet metrics dashboard for telemetry all nodes need
   # include puppet_metrics_dashboard::profile::master::install
   ###################################################################
 
   include puppet_metrics_dashboard::profile::master::install
 
-
-  #####################3. Metrics Dashboard postgres access ############
-  # Determine if node is pe_postgres host and conditionally apply include puppet_metrics_dashboard::profile::master::postgres_access
-  ######################################################################
-
-  #The following code serves to check that postgres is present and then declares the class
-
-  if $facts['pe_postgresql_info'] != undef {
-    include puppet_metrics_dashboard::profile::master::postgres_access
-  }
-
-
-
   #####################3. RSANpostgres command access ######################
   # Determine if node is pe_postgres host and conditionally apply Select Access for the RSAN node cert to all PE databases
-  # Hint metrics dashboard postgres access code can be duplicated and repurposed
+  # and conditionally apply include puppet_metrics_dashboard::profile::master::postgres_access
   ######################################################################
 
-  if $facts['pe_postgresql_info'] != undef {
+  if $facts['pe_postgresql_info'] != undef and $facts['pe_postgresql_info']['installed_server_version'] != '' {
```

Comment:
Wait on compilers `pe_postgresql_info` and `installed_server_version` is an *empty string*? What a pain in the ass. I was going to suggest:

```puppet
if $facts.dig('pe_postgresql_info', 'installed_server_version') {
```

but that wouldn't work of the fact is returning an empty string

Reply:
Yeah, I had initially thought it was undef, but it turned out to be a string. The whole fact should be undef, but thats a PE bug we need to workaround.

```
Notice: Scope(Class[main]): String
Notice: Scope(Class[main]): 
```

Reply:
Yeah what a pain. Good catch though